### PR TITLE
Assume Apache 2.0 license for vendor/nestlabs modules

### PR DESCRIPTION
**Pull request by samhansen**
This assumes the Apache 2.0 license for the as-opensourced modules found
in src/lib/profiles/vendor/nestlabs.

### CODE REVIEW

File: src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp [hunk 1]
```diff
@@ -1,15 +1,18 @@
 /*
- *
- *    Copyright (c) 2014-2018 Nest Labs, Inc.
+ *    Copyright (c) 2013-2019 Nest Labs, Inc.
```
Comment (robszewczyk) [inline src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp]: Copyright (c) 2014-2018 Nest Labs, Inc.
Copyright (c) 2019 Google, LLC.

File: src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.cpp [hunk 2]
```diff
@@ -1,15 +1,18 @@
 /*
- *
- *    Copyright (c) 2016 Nest Labs, Inc.
+ *    Copyright (c) 2013-2019 Nest Labs, Inc.
```
Comment (robszewczyk) [inline src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.cpp]: Copyright (c) 2016 Nest Labs, Inc.
    Copyright (c) 2019 Google, LLC.

File: src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.h [hunk 3]
```diff
@@ -1,15 +1,18 @@
 /*
- *
- *    Copyright (c) 2016 Nest Labs, Inc.
+ *    Copyright (c) 2013-2019 Nest Labs, Inc.
```
Comment (robszewczyk) [inline src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.h]: Copyright (c) 2016 Nest Labs, Inc.
    Copyright (c) 2019 Google, LLC.

File: src/lib/profiles/vendor/nestlabs/thermostat/IfjStatusStr.cpp [hunk 4]
```diff
@@ -1,15 +1,18 @@
 /*
- *
- *    Copyright (c) 2016 Nest Labs, Inc.
+ *    Copyright (c) 2013-2019 Nest Labs, Inc.
```
Comment (robszewczyk) [inline src/lib/profiles/vendor/nestlabs/thermostat/IfjStatusStr.cpp]: Copyright (c) 2016 Nest Labs, Inc.
    Copyright (c) 2019 Google, LLC.

File: src/lib/profiles/vendor/nestlabs/thermostat/NestThermostatWeaveConstants.h [hunk 5]
```diff
@@ -1,15 +1,18 @@
 /*
- *
- *    Copyright (c) 2013-2014 Nest Labs, Inc.
+ *    Copyright (c) 2013-2019 Nest Labs, Inc.
```
Comment (robszewczyk) [inline src/lib/profiles/vendor/nestlabs/thermostat/NestThermostatWeaveConstants.h]: Copyright (c) 2013-2014 Nest Labs, Inc.
    Copyright (c) 2019 Google, LLC.

File: src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp (hunk 1 above)
Comment (samhansen) [reply src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp]: Done.

File: src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.cpp (hunk 2 above)
Comment (samhansen) [reply src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.cpp]: Done.

File: src/lib/profiles/vendor/nestlabs/thermostat/IfjStatusStr.cpp (hunk 4 above)
Comment (samhansen) [reply src/lib/profiles/vendor/nestlabs/thermostat/IfjStatusStr.cpp]: Done.

File: src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.h (hunk 3 above)
Comment (samhansen) [reply src/lib/profiles/vendor/nestlabs/dropcam-legacy-pairing/DropcamLegacyPairing.h]: Done.

File: src/lib/profiles/vendor/nestlabs/thermostat/NestThermostatWeaveConstants.h (hunk 5 above)
Comment (samhansen) [reply src/lib/profiles/vendor/nestlabs/thermostat/NestThermostatWeaveConstants.h]: Done.

**Pull request merged**